homeassistant/components/sleepiq/manifest.json

```diff
@@ -3,5 +3,5 @@
   "name": "SleepIQ",
   "documentation": "https://www.home-assistant.io/integrations/sleepiq",
   "requirements": ["sleepyq==0.8.1"],
-  "codeowners": []
+  "codeowners": ["@home-assistant", "@Jay2645", "@Jpeterson37"]
```

Comment:
Please remove `@home-assistant`. Is @Jay2645 aware of you making him codeowner?

Reply:
So I've seen, but is he aware he is becoming a codeowner and will be notified of issues raised by endusers for this integration?